bluesky/utils.py

```diff
@@ -40,7 +40,7 @@ def __new__(cls, command, obj=None, *args, run=None, **kwargs):
         return super(Msg, cls).__new__(cls, command, obj, args, kwargs, run)
 
     def __repr__(self):
-        return f"{self.command}[{self.run}]: ({self.obj}), {self.args}, {self.kwargs}"
+        return f"{self.command}({self.obj}), {self.args}, {self.kwargs}"
```

Comment:
This looks like a mistake during the rebase. This line should not be changed.

Reply:
This is base on function-call format suggestion [here](https://github.com/bluesky/bluesky/issues/1185#issuecomment-497014753).
It's feature change. I feel it's better. However, I don't have strong opinion about this.

Reply:
That suggestion was made before a `run` attribute was added to `Msg`. It doesn't seem right to just delete/hide `self.run`. What about
```suggestion
        return f"{self.command}({self.obj}), {self.args}, {self.kwargs}, run={self.run}"
```

The function-call format is nice but could be misleading because

```py
Msg('set', device, timeout=10)
```

will render as

```py
set(device, {'timeout': 10})
```

when in fact what is being executed is more like

```py
set(device, timeout=10)
```

I'm not sure if that matters. cc @klauer @tacaswell

Reply:
Looking more closely at @klauer's suggestion, @ke-zhang-rd, it looks like he noticed that issue too. His suggestion includes unpacking notation `*()` and `**{}` in the log message.

Reply:
Yes, I (originally) was worried about misrepresenting how args/kwargs were being passed in. I think it's important to be clear about that.

Whether it ends up being perfectly formatted as a syntactically clean function call or using `*`, `**`, it doesn't so much matter to me - though my preference would differ depending on who would be looking at these messages (end-users vs developers).

Reply:
Thanks guys, double check, because of rebase work, I may manually edit this code to
```
return f"{self.command}({self.obj}, *{self.args}, **{self.kwargs}, run={self.run})"
```
It looks like
```
[D 14:54:59.003      run_engine: 1386] stage(SynGauss(name='det1', value=5.0, timestamp=1582055698.7999759), *(), **{}, run=None)
```